### PR TITLE
tools/file_packager.py: Cleanup code and and its generated output. NFC

### PR DESCRIPTION
**Pull request by sbc100**
- Use enumerate() to avoid a counter
- (mostly) Fix indentation of generated JS

### CODE REVIEW

**Comment by sbc100** [comment]
View with "hide whitespace" since this diff has a bunch of white-space only changes to fix indentation.

File: tools/file_packager.py [hunk 1]
```diff
@@ -282,8 +282,7 @@ def main():
   # standalone calls
   if not from_emcc:
     ret = '''
-  var Module = typeof %(EXPORT_NAME)s !== 'undefined' ? %(EXPORT_NAME)s : {};
-  ''' % {"EXPORT_NAME": export_name}
+  var Module = typeof %(EXPORT_NAME)s !== 'undefined' ? %(EXPORT_NAME)s : {};\n''' % {"EXPORT_NAME": export_name}
```
[the remark this answers is not in the document]
Comment (kripken) [reply tools/file_packager.py:285]: > The three spaces IIUC were there to avoid the """ being in column zero or the python code.

Can we just remove the three spaces then? Python accepts that IIANM.

[the remark this answers is not in the document]
Comment (sbc100) [reply tools/file_packager.py:285]: Yes, but the problem is that we end up with lots of stuff in column 1 of the python code.. which IMHO makes it harder to read in general.. and add a lot of extra lines to the lines don't do anything.

I've updated the this PR to be even more consistent and added quick test for trailing/extra whitespace in the output.

[the remark this answers is not in the document]
Comment (kripken) [reply tools/file_packager.py:285]: sgtm, I'm mostly neutral here, so I don't want to block this. Good idea about the test.

**Review by kripken — APPROVED** [review]

**Comment by sbc100** [comment]
This is mostly just cleanup to make things more readable for me.. in preparation for actual file_packager changes to follow.
